nbgrader/exchange/collect.py

```diff
@@ -1,7 +1,9 @@
 import os
 import glob
+import pwd
```

Comment:
Ah yeah, I think you will need to wrap this in a try/except so that the file can at least be imported on windows.

Reply:
I think you still need to remove this line as it's still trying to import `pwd` on windows :(

Reply:
Hem, hem, hem :-) Sounds like I was not quite awake. Thank you for the cleanup!

Reply:
It happens 😄No problem!